shub/image/init.py

```diff
@@ -22,42 +21,42 @@
 """.format(docker_app_dir=DOCKER_APP_DIR)
 
-BASE_SYSTEM_DEPS = [
-    'telnet', 'vim', 'htop', 'strace', 'ltrace', 'iputils-ping', 'lsof'
-]
 BASE_PYTHON_DEPS = [
-    'BeautifulSoup==3.2.0', 'MySQL-python==1.2.3', 'Scrapy==1.0.3',
-    'Twisted==11.1.0', 'boto==2.28.0', 'guppy==0.1.10', 'hubstorage==0.19.1',
-    'ipython==3.2.1', 'lxml==3.4.4', 'numpy==1.6.1', 'pika==0.9.14',
-    'psycopg2==2.4.5', 'pymongo==2.6.3', 'queuelib==1.2.2',
-    'raven==5.0.0', 'requests==2.5.3', 'scrapely==0.12.0',
-    'scrapinghub==1.7.0', 'scrapylib==1.6.0', 'setproctitle==1.0.1',
-    'slybot==0.12.1', 'w3lib==1.13.0',
-    'scrapinghub-entrypoint-scrapy==0.6.0'
+    'BeautifulSoup==3.2.0',
+    'MySQL-python==1.2.3',
+    'guppy==0.1.10',
+    'ipython==5.3.0',
+    'numpy==1.12.1',
+    'pika==0.10.0',
+    'psycopg2==2.7.1',
+    'pymongo==3.4.0',
+    'setproctitle==1.1.10',
+    'slybot==0.12.1',
 ]
 
-SHORT_HELP = "Form Dockerfile for a given Scrapy project and save it."
+SHORT_HELP = "Create Dockerfile for existing Scrapy project."
 
 HELP = """
-Init command creates a Dockerfile for your project. It's likely that
-default values will fit to you, otherwise you will be able to edit your
-Dockerfile as you want.
+Init command creates a Dockerfile for existing Scrapy project. This tool is for users
+who want to create a custom Docker image and don't have a Dockerfile yet. If generated
+Dockerfile doesn't fit your project feel free to edit it.
+
+Python packages
+
+If there's a requirements.txt file in the project directory - it will be added to the
+Dockerfile. Also it's possible to provide a path to requirements file via --requirements
+option. Otherwise new requirements.txt file will be created in the project directory
+with the recommended Python packages. Use --list-recommended-reqs to list them.
 
-System deps for Dockerfile:
-By default there're several system deps to be included to the Dockerfile ({}),
-you can extend it via --add-deps option, or redefine at all with --base-deps
-option.
+It's recommended to include scrapinghub-entrypoint-scrapy package - it is a
+support layer that passes data from the job to Scrapinghub storage. Otherwise
+you will need to send data to Scrapinghub storage using HTTP API.
 
-Python deps for Dockerfile:
-The correct way to install python deps is using requirements.txt. If there's
-no requirements.txt in the project_dir folder and no provided value for it
-via --requirements option, we'll create new requirements.txt in the
-project_dir with the recommended deps(use --list-recommended-reqs to list it).
+System packages
 
-You should also include scrapinghub-entrypoint-scrapy package to your py-reqs,
-it's a necessary condition to run your project with Kumo (use 'shub image test'
-to check if your image fits our expectations).
-""".format(BASE_SYSTEM_DEPS)
+You can extend list of system packages installed in the image via --add-deps option or
+provide a comma-separated list of packages via --base-deps option.
```

Comment:
`--add-deps` is also a comma-separated list, right? Let's remove `--base-deps` from the help message if we deprecated it.